nginx.conf

```diff
@@ -35,6 +35,7 @@ http {
     location / {
       proxy_pass <%= ENV["FEDERALIST_S3_BUCKET_URL"] %>/;
       proxy_redirect ~*^/site/([^/]+)/([^/]+)/(.*) " /$3";
+      proxy_redirect ~*^/preview/([^/]+)/([^/]+)/([^/]+)/(.*) " /$4";
```

Comment:
For reference, preview URIs start like this: `/preview/:owner/:repo/:branch`

`~*^/preview/([^/]+)/([^/]+)/([^/]+)/(.*) " /$4"` should redirect to whatever's after the preview base URI.